ui/perfherder/compare/CompareTable.jsx

```diff
@@ -172,7 +176,15 @@ export default class CompareTable extends React.Component {
         <thead>
           <tr className="subtest-header bg-lightgray">
             <th className="text-left">
-              <span>{testName}</span>
+              <span>
+                {testName}
+                {frameworkName === 'talos' && (
+                  <a href={url} style={{ color: '#23527c' }}>
```

Comment:
we should have the style defined in `perf.css` file.

https://github.com/mozilla/treeherder/blob/master/ui/css/perf.css#L312

```
.compare-table .result-links a,
.compare-table .subtest-header a {
  color: #23527c;
}
```